symfony/framework-bundle/3.3/post-install.txt

```diff
@@ -4,7 +4,7 @@
 
   * <fg=blue>Run</> your application:
     1. Change to the project directory
-    2. Execute the <comment>php -S 127.0.0.1:8000 -t public</> command;
+    2. Execute the <comment>php -S 127.0.0.1:8000 public/index.php</> command;
```

Comment:
I think we still need to set the document root to `public`, otherwise loading assets won't work.

The server:start command does not use the option, because the default behavior of PHP is to use the CWD as document root, and the command runs the process with the `public` folder as working directory.

Reply:
Yea, I just had a user more or less hit this issue I think.